tests/test_cli_paths.py

```
@@ -0,0 +1,76 @@
+
+from pathlib import Path
+from types import SimpleNamespace
+from unittest.mock import patch
+
+from mmrelay.cli import handle_config_command, handle_paths_command
+from mmrelay.constants.app import APP_DISPLAY_NAME, APP_NAME
+
+
+def test_handle_paths_command_basic(capsys, monkeypatch):
+    """Test handle_paths_command prints basic info."""
+    home = Path("/tmp/fake_home")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake_home"

(S108)